mixer/pkg/runtime2/routing/monitoring.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package routing
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	meshFunction = "meshFunction"
+	handlerName  = "handler"
+	adapterName  = "adapter"
+	errorStr     = "error"
```

Comment:
non-blocking: @mandarjog considering that (iirc) this field is used elsewhere for other monitoring bits, should we considering having a mixer-specific (or perhaps Istio-wide) set of monitoring consts for labels, etc., to ensure consistent labeling?

Reply:
Yes, I agree we should have istio-wide labels would be great.
Also also `/pkg/monitoring` perhaps with const labels and these types of convenience functions. **not-blocking**